### ISSUE TITLE
Classes that import io.kotest are not considered a test case

### ISSUE THREAD

**Issue by martinvisser** [pinterest/ktlint#2289]
## Expected Behavior

With the promotion of function naming I now see errors like "Function name should start with a lowercase letter (except factory methods) and use camel case" even though this is within a test. In our test case we use Kotest and no `org.junit` is used. Expected behavior would be that these functions are also considered as valid, because it's a test case. 

## Current Behavior

The following test case is considered to have an invalid function name. According to [Function naming](https://pinterest.github.io/ktlint/1.0.0/rules/standard/#function-naming) tests are considered to have an import of `org.junit`, `org.testng` or `kotlin.test`, but this example doesn't have any. This is partially due to the choice of using Kotest, but also because this is part of a maven plugin that extends from rather ancient test style nearly forced upon by the Maven Plugin Testing Harness.

```kotlin
package some.package

import io.kotest.matchers.collections.shouldContainExactly
import io.kotest.matchers.maps.shouldContainKey
import kotlin.io.path.readLines

class AgentMojoTest : BaseMojoTest() {
    fun `test should write arg line`() { // considered as invalid
    }
}
```

```kotlin
package some.package

import org.apache.maven.plugin.testing.AbstractMojoTestCase

abstract class BaseMojoTest : AbstractMojoTestCase() {
}
```

## Additional information
* Current version of ktlint: `1.0.0`

**Comment by paul-dingemans** [comment]
Indeed "io.kotest" is not on the ignore list in https://github.com/pinterest/ktlint/blob/a679dd4f2e84288e98d6d06a42091bcf0b0087a8/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/rules/FunctionNamingRule.kt#L56. It is an easy fix though.
